apps/ledger-live-mobile/src/screens/SyncOnboarding/FirmwareUpdateDrawer.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import {
+  BottomDrawer,
+  BoxedIcon,
+  Button,
+  Flex,
+  Text,
+} from "@ledgerhq/native-ui";
+import { NanoFirmwareUpdateMedium } from "@ledgerhq/native-ui/assets/icons";
+import { useTranslation } from "react-i18next";
+
+export type Props = {
+  isOpen: boolean;
+  onSkip?: () => void;
+  onUpdate?: () => void;
+  onClose?: () => void;
+  productName: string;
+};
+
+const FirmwareUpdateDrawer = ({
+  isOpen,
+  onSkip,
+  onUpdate,
+  onClose,
+  productName,
+}: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <BottomDrawer
+      onClose={() => {
+        onClose?.();
+        onSkip?.();
```

Comment:
:wat: I didn't know this syntax was working haha! Noice
Regardless, can you move this to a useCallback?

Reply:
i added the useCallback on this one

Reply:
as `BottomDrawer` is not a memoized component, it will be re-rendered each time `LanguageSelect` re-renders, even if its props did not change. So it won't change anything to memoized this function